src/zino/tasks/linkstatetask.py

```diff
@@ -51,6 +57,20 @@ async def run(self):
 
         self._update_interfaces(attrs)
 
+    async def poll_single_interface(self, ifindex: int):
+        """Polls and updates a single interface"""
+        snmp = SNMP(self.device)
+        poll_list = [("IF-MIB", column, str(ifindex - 1)) for column in BASE_POLL_LIST]
+        result = await snmp.getnext2(*poll_list)
+        self.sysuptime = await self._get_uptime(snmp)
+        _logger.debug("poll_single_interface %s result: %r", self.device.name, result)
+
+        assert all(ident.index == OID(f".{ifindex}") for ident, value in result)
+        row = {ident.object: value for ident, value in result}
+
+        self._update_single_interface(row)
+        return row
```

Comment:
Is this actually supposed to return something? I cant see the return value being used anywhere

Reply:
Naw. I might have kept it there for testing and forgotten to remove it :)